hs_file_types/models/base.py

```diff
@@ -775,7 +775,7 @@ def create_aggregation(cls, dataset_name, resource, res_files=None, new_files_to
             uploaded_file = UploadedFile(file=open(f, 'rb'), name=os.path.basename(f))
 
             new_res_file = add_file_to_resource(
-                resource, uploaded_file, folder=folder_path, add_to_aggregation=False
+                resource, uploaded_file, folder=folder_path, add_to_aggregation=False, save_file_system_metadata=True
```

Comment:
nice, we generate system_metadata when we create an aggregation.

Some other places to check:

What about here in a timeseries?
https://github.com/hydroshare/hydroshare/blob/5182-store-file-checksum-modified-time-in-db/hs_file_types/models/timeseries.py#L2602

What about here (when ingesting a bag):
https://github.com/hydroshare/hydroshare/blob/5182-store-file-checksum-modified-time-in-db/hs_core/management/commands/ingest_bag.py#L103

And here:
https://github.com/hydroshare/hydroshare/blob/5182-store-file-checksum-modified-time-in-db/hs_file_types/models/base.py#L1142

Reply:
Thanks for catching these. Fixed it now.

Reply:
great!